app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -94,12 +96,15 @@ class AdministratorControlsActivityPresenter @Inject constructor(
       ) as AdministratorControlsFragment?
   }
 
-  /** Loads the profile list fragment as the [AdministratorControlsActivity] is started in multipane tablet mode. */
+  /**
+   * Loads the profile list fragment as the [AdministratorControlsActivity]
+   * is started in multipane tablet mode.
```

Comment:
This should be wrapped at 100 characters.

Reply:
done